Testing/ParticlesTests/ParticlesTests.cpp

```diff
@@ -27,6 +28,23 @@ const std::vector<std::string> filenames = {
   test_dir + "seg.ellipsoid_21.isores.pad.com.aligned.cropped.tpSmoothDT_world.particles"
 };
 
+TEST(ParticlesTests, vnlVector)
```

Comment:
Verification of results should be added to the test in order to ensure underlying functionality of the component being tested remains consistent.

Reply:
Done